docs/build/build-protocol-info.md

```diff
@@ -224,8 +224,25 @@ transaction data using the
 
 ## Runtime Upgrades
 
-Polkadot allows [runtime upgrades](../learn/learn-runtime-upgrades.md) without a hard fork. The runtime is a
```

Comment:
I think it'd be useful to add the link in so people can go to more detailed info if they want. Otherwise all good.